app/models/user.rb

```diff
@@ -5,6 +5,52 @@ class User < ApplicationRecord
   include Searchable
   include Storext.model
 
+  # NOTE: @citizen428 This is temporary code during profile migration and will
+  # be removed.
+  # rubocop:disable Metrics/BlockLength
+  concerning :ProfileMigration do
+    included do
+      PROFIE_FIELDS =
+        (Profiles::ExtractData::DIRECT_ATTRIBUTES + Profiles::ExtractData::MAPPED_ATTRIBUTES.values).freeze
+
+      default_scope { includes(:profile) } # rubocop:disable Rails/DefaultScope
```

Comment:
In this PR I apparently use everything I usually rant about. 😒 That said, we do want to always load profiles with users and for the time being and given the temporary nature of this code, this seems to be an ok solution. I'll look into how to tie this into Devise/Warden/`current_user` eventually.

Reply:
I'm cool with this bc its temporary. Lets get the job done quickly and smoothly then we can simply remove the code.

Reply:
Cool @mstruve, thanks for confirming!

> Lets get the job done quickly

That's the goal. Though I'm not going to rush this, as I don't want to risk messing up live data on DEV. 

I had some time to think about this over the weekend (cc @Ridhwana): relatively soon we want to add code for updating profile data through the profile page but using the new model. At that point, we can reverse the direction of the sync.

Before: Changes update `User` first, syncs to `Profile`.
After: Changes update `Profile` first, but syncs back to `User`.

Once we're confident that everything works as expected, we can remove the sync and drop the related columns from `users`, but I generally wait for a bit before performing this sort of destructive action on production data.

I have some ideas how to deal with users vs profiles, preloading, etc, after, but first I want to see how our new use case looks like since according to a discussion with @benhalpern on Forem.dev (see [here](https://forem.dev/ben/comment/9p) we may remove some things from the UI to simplify things overall.